### PR TITLE
arch/mips64{,r2}el: add revised support

### PR DESCRIPTION
**Pull request by lmy441900**
These are changes about autobuild3 for the rebooted AOSC OS MIPS port.

The new AOSC OS MIPS port has 2 variants:

1. **MIPS64EL** ("Care" Port)
  - Target ISA: **MIPS-III**
  - Target ABI: **N64, hard-float**
  - Main Supporting Platform: **Loongson 2F** Machines (including Yeelong Netbooks, Fulong Mini-PCs)
2. **MIPS64R2EL** ("Modern" Port)
  - Target ISA: **MIPS64 revision 2**
  - Target ABI: **N64, hard-float**
  - Main Supporting Platform: **Loongson 3** Machines

> [Forwarded from imi415]
> aosc os，关爱您、您的开发板、您的谜之处理器和您的史前遗产

### CODE REVIEW

File: arch/mips64r2el.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 ##arch/mips64el.sh: Build definitions for mip64r2el (optimized for Loongson 3A).
```
Comment (lmy441900) [inline arch/mips64r2el.sh]: Ayy

File: arch/mips64el.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+##arch/mipsel.sh: Build definitions for mipsel (optimized for Loongson 2F).
```
Comment (lmy441900) [inline arch/mips64el.sh]: Ayy

**Comment by MingcongBai** [comment]
@FlyGoat Could you take a look at this so that we can go ahead with merging this?

**Comment by FlyGoat** [comment]
@MingcongBai I have no objections to that.

**Comment by KexyBiscuit** [comment]
Seems to be superseded by #110.

**Pull request closed without merging**